packages/design/src/colors.css

```diff
@@ -170,5 +170,5 @@
   --color-indigo--dark: rgb(55, 69, 132);
 
   --color-white--rgb: 255, 255, 255;
-  --color-white: rgba(var(--color-white--rgb));
+  --color-white: rgb(var(--color-white--rgb));
```

Comment:
This spooks me because I have no idea how this was working before... I assume if RGBA doesn't get an argument for `a` it falls back to 100%?